### DISCUSSION TITLE
Chart source with multiple columns

### DISCUSSION THREAD

**Discussion by lstillwe** [TerriaJS/terriajs#6699] [Q&A]
I want to create a Chart from a user click on a WFS feature. One of the features is a csv url link to a csv file that looks like this ...
```
TIME, Value1, Value2
2021-11-27 00:00:00,0.22,0.151
```
I have the chart working with this FeatureInfoTemplate definition ...
```
"featureInfoTemplate": {
    "template": "<div><chart src={{csvurl}}></chart></div>"
 }
```
However, only Value1 is plotted on the chart. Is only the first Value column supported for plotting a line chart? Do I have to have a separate source for each Value I would like to plot?

Thanks very much!

**Comment by lstillwe** [comment]
UPDATE: Realized that Value2 is available when expanding the chart popup to the bottom of the map. Is there a way to make all values show up in the plot automatically? 

Thanks again!

**Comment by tephenavies** [comment]
Hi @lstillwe. I think this issues describes your problem https://github.com/TerriaJS/terriajs/issues/6434. I have no estimate for when this could be fixed.
